app/jobs/webhook/dispatch_event_job.rb

```diff
@@ -0,0 +1,10 @@
+module Webhook
+  class DispatchEventJob < ApplicationJob
+    queue_as :dispatch_webhook_events
```

Comment:
just a little detail, maybe this should be called `webhook_dispatch_events`, it'd be easier to find it later on in the UI as well.